CMakeLists.txt

```diff
@@ -411,8 +411,11 @@ if(WITH_CORE)
     message(STATUS "Qt WebKit support DISABLED.")
   endif()
 
+  set (WITH_INTERNAL_LAZPERF FALSE CACHE BOOL "Determines whether LazPerf should be built from internal copy")
```

Comment:
We should default to internal lazperf

Reply:
If it's ok for you, i'll be very happy to use the default one! :)